Update customer-support.md (#172)
mattermost/mattermost-handbook: operations/business-operations/customer-support.md

> Details of what the Support Team provides for our Enterprise Customers can be found here, this includes our support options and SLA’s.

Reviewer: ```suggestion
Details of what the Support Team provides for our Enterprise Customers can be found here, this includes our support options and SLAs.
```

**Meet the Team**

![](../../.gitbook/assets/Support-team-image-for-handbook.png)

**Where to find us**

* Mattermost Channels: Customer Support, Support Standup
* Support Team Email: support@mattermost.com
* Customer Support Portal: https://support.mattermost.com

**Mattermost Ticketing System**

> To ensure that all issues raised with the Mattermost Support Team can be easily tracked, updated, and resolved, the Support Team uses [Zendesk](https://www.zendesk.com/) as their case management software.

Reviewer: ```suggestion
To ensure that all issues raised with the Mattermost Support Team can be easily tracked, updated, and resolved the Support Team uses [Zendesk](https://www.zendesk.com/) to manage cases.
```

Customers can access the Support Portal and submit/view their tickets via https://support.mattermost.com. 

**New Users to Mattermost Support**

If you are new to Mattermost Support you can [click here](https://mattermost.zendesk.com/auth/v2/login/registration?auth_origin=327479%2Ctrue%2Ctrue&brand_id=327479&return_to=https%3A%2F%2Fsupport.mattermost.com%2Fhc%2Fen-us&theme=hc) to set up and verify your account.

**Existing Users of Mattermost Support**

> If you’ve worked with the Mattermost Support Team before and have existing or previous tickets with Support then you already have an account with the email address that you would have used.  To (re)set your password, you can [click here.](https://mattermost.zendesk.com/auth/v2/login/password_reset?auth_origin=327479%2Ctrue%2Ctrue&brand_id=327479&return_to=https%3A%2F%2Fsupport.mattermost.com%2Fhc%2Fen-us&theme=hc)

Reviewer: ```suggestion
If you’ve worked with the Mattermost Support Team before and have existing or previous tickets with Support then you already have an account with the email address that you would have used.  To (re)set your password, you can [click here](https://mattermost.zendesk.com/auth/v2/login/password_reset?auth_origin=327479%2Ctrue%2Ctrue&brand_id=327479&return_to=https%3A%2F%2Fsupport.mattermost.com%2Fhc%2Fen-us&theme=hc).
```
